target/snitch_cluster/cfg/snax-gemm.hjson

```diff
@@ -86,7 +86,7 @@
         xf8alt: true,
         xfdotp: true,
         xfvec: true,
-        snax_acc: 2,
+        snax_acc: "snax_gemm_wrapper",
```

Comment:
nit: the other one is not named *_wrapper

Reply:
I know 😭 but this needs to call the module name of the accelerator. I can add the `wrapper` thing for the HWPE. Wait.